java/daikon/Chicory.java

```diff
@@ -113,6 +113,12 @@
   @Option("Number of calls after which sampling will begin")
   public static int sample_start = 0;
 
+  @Option("Should Exception Handling be taken care of?")
+  public static boolean exception_handling = false;
+
+  @Option("RemoteDebug Test")
```

Comment:
Rather than creating a variable whose only purpose is to be copied into another variable, I suggest that you put `@Option` on the existing `RemoteDebug` variable and rename it to `remote_debug`.